cockatrice/src/setsmodel.h

```diff
@@ -43,11 +43,11 @@ class SetsModel : public QAbstractTableModel {
     QStringList mimeTypes() const;
     void swapRows(int oldRow, int newRow);
     void toggleRow(int row, bool enable);
-    void toggleAll(bool enable);
+    void toggleRow(int row);
+    void toggleAll(int row);
```

Comment:
toggleAll will take a row and change all sets to copy that row's status (So if you pass it a row that's enabled, it will enable everything and if you pass it a disabled row, it will disable everything)